tests/create-and-vary-a-licence-and-delius/view-case.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from '@playwright/test'
+import * as dotenv from 'dotenv'
+import { login as cvlLogin } from '../../steps/cvl-licences/login'
+import { data } from '../../test-data/test-data'
+
+dotenv.config() // read environment variables into process.env
+
+test('View case in Create and Vary a Licence', async ({ page }) => {
+    test.skip() // This test doesn't do anything yet, it just proves connectivity. We should write a proper test when we do the CVL integration.
```

Comment:
should the skip be left in?

Reply:
Yeah I think so.  I didn't want this noddy test to be enough to deploy to prod, we should add a proper end-to-end test in as part of completing the actual integration work.